volatility3/framework/plugins/windows/pslist.py

```diff
@@ -197,7 +198,7 @@ def list_processes(
         filter_func: Callable[
             [interfaces.objects.ObjectInterface], bool
         ] = lambda _: False,
-    ) -> Iterable[interfaces.objects.ObjectInterface]:
+    ) -> Iterator["extensions.EPROCESS"]:
```

Comment:
Is there a reason to put this in quotes?  It would've been imported already by now, wouldn't it?

Reply:
I thought it would have, but ran into this after making the initial change: `AttributeError: partially initialized module 'volatility3.framework.symbols.windows.extensions' has no attribute 'EPROCESS' (most likely due to a circular import)`

Reply:
Ok, quotes it is then...  5:)